packages/api/src/graphql/peer-article/peer-article.private-queries.ts

```diff
@@ -51,13 +58,16 @@ export const getAdminPeerArticles = async (
           fieldName: 'articles',
           args: {
             cursor: cursors ? cursors[peer.id] : undefined,
-            take: 50,
+            take: articleToTakeFromEachPeer,
+            // Skip isn't backwards compatible since the in pre primsa it means skip pages and in prisma skip object.
+            skip: articleToSkipFromEachPeer,
```

Comment:
wouldn't this now break pagination for tsüri etc?

Reply:
Its broken anyways....  Since the skip logic has been changed by prisma migration... So its not possible to make to not break it in my mind or do you have a better idea where we limit the load on the peering and keep the pagination for all medias working?

Reply:
Should we merge https://github.com/wepublish/wepublish/pull/711/files this then also into this PR? This actually makes pagination in the UI available too and fixes the pages that are too many :D

Reply:
Ah sounds great. Yes please merge this if you like but not here please to the https://github.com/wepublish/wepublish/pull/828 pr :-) Beacause this one we dont merge because of to many conflicts :-)